File: DofusSpa/src/app/_models/equipment-recipe.ts
import { Equipment } from "./equipment";

export interface EquipmentRecipe{
    id : number,
    ankamaID : number,
    name : string,
    url : string,
    imgUrl : string,
    quantity : number,
    level : number,
    type: string,
    equipmentID : number,
    equipment : Equipment
}

File: DofusApi/Models/EquipmentRecipe.cs
namespace DofusApi.Models
{
    public class EquipmentRecipe
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string ImgUrl { get; set; }
        public int Quantity { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public int EquipmentID { get; set; }
        public Equipment Equipment { get; set; }
        
        
    }
}